### ISSUE TITLE
Collaborators can also add captions to the owner's pictures, as well as notifications

### ISSUE THREAD

**Issue by reaiwaris** [ente-io/photos-desktop#261]
Collaborators can fill in or comment (rate) the pictures shared by the owner, and collaborators can also receive some notifications when the owner updates the gallery, I think there are only notifications when the shared pictures are added，That's the suggestion above, but it feels like it takes a little time to implement

**Comment by vishnukvmd** [comment]
> Collaborators can fill in or comment (rate) the pictures shared by the owner

Related: https://github.com/ente-io/photos-app/issues/1468.

>  collaborators can also receive some notifications when the owner updates the gallery

If you're part of a shared album, and someone other than you adds photos, you should already be receiving a notification when the app syncs in the background. Please let us know if this isn't the case!

**Comment by reaiwaris** [comment]
Yes, the owner was notified when it was updated, but I prefer to express the ability for collaborators to write descriptions for other people's photos, or comment on pictures, and hopefully that will be implemented
